packages/material-ui-lab/src/CalendarPicker/PickersCalendar.tsx

```diff
@@ -58,6 +57,9 @@ export interface PickersCalendarProps<TDate> extends ExportedCalendarProps<TDate
   TransitionProps?: Partial<SlideTransitionProps>;
 }
 
+const weeksContainerHeight = (DAY_SIZE + DAY_MARGIN * 4) * 6;
+
+// TODO remove PickersCalendarClassKey in CalendarPickerSkeleton migration
 export type PickersCalendarClassKey =
```

Comment:
Cannot remove `PickersCalendarClassKey` and `styles` in this PR due to dependency to `CalendarPickerSkeleton`. will removed them in CalendarPickerSkeleton migration PR